frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -15,14 +15,14 @@ export let AddDatasetDrawer = withStore('selectedDataset')(({ }) =>
   <div className='AddDatasetDrawer'>
     <BackLink />
     <h2 className='Secondary'>Upload your data to explore</h2>
-    <p className='MediumWeight Muted'>
+    <span className='MediumWeight Muted'>
```

Comment:
What is the difference?

Reply:
I saw this when running the app locally:

<img width="997" alt="screen shot 2018-03-04 at 12 25 00 pm" src="https://user-images.githubusercontent.com/1761758/36950184-2cf0fcbc-1fa7-11e8-941e-03a19f700ab2.png">

Btw, how have you guys been testing it? It looks like the local version of the app has been broken since we introduced stacked bar charts.

Reply:
Interesting! I wasnt seeing this. We are only running with docker, but I remember @akegan mentioning that she had issue with the local version.

Reply:
yeah I got a different issue with 'cannot find chart' when running locally with the stacked bar charts =(. haven't prioritized fixing it yet

Reply:
See comment below!